docs/modules/i3s/recipes/attribute-driven-colorization.mdx

```diff
@@ -0,0 +1,130 @@
+# Attribute-driven colorization
+
+import Demo from 'examples/website/i3s-colorization-by-attributes/src/app';
+
+<div style={{height: '50vh'}}>
+  <Demo />
+</div>
+Please find source code of the example [here](https://github.com/visgl/loaders.gl/tree/master/examples/website/i3s-colorization-by-attributes)
+
+In I3S, a feature represents a real-world object within a node. For example, a building within a 3D object scene layer. Node resources such as geometry buffer and attributes can belong to a feature and can be accessed by an object-ID.
+
+In terms of geometry, every vertex of a mesh is associated with some feature. At the same time, every feature is associated with feature attribute values. For example, there might be `HEIGHT` attribute that stores roof height information about every building.
+
+All that means that it is possible to make some visual effects related to attribute value. It might be text labels, colors, opacity etc.
+
+The complete case of attributes colorization is done in [I3S Explorer](https://i3s.loaders.gl/viewer?tileset=new-york). It is an open source ReactJS application. See source code on [GitHub](https://github.com/visgl/loaders.gl-showcases).
+
+![I3S New York dataset colorized by height roof](i3s-new-york-colorize-by-heightroof.png)
+
+## Find out layer's attributes
+
+It is necessary to pick some attribute to colorize a layer by. So it is necessary to load the layer JSON:
+
+```javascript
+import {load} from '@loaders.gl/core';
+import {I3SLoader} from '@loaders.gl/i3s';
+
+const i3sLayer = await load(url, I3SBuildingSceneLayerLoader);
+```
+
+List and types of attributes might be taken from [`i3sLayer.fields`](https://github.com/Esri/i3s-spec/blob/master/docs/1.9/field.cmn.md) and [`i3sLayer.attributeStorageInfo`](https://github.com/Esri/i3s-spec/blob/master/docs/1.8/attributeStorageInfo.cmn.md) properties.
+
+## Setup colorization scale
+
+Attributes colorization capability applies linear color gradient. To create this gradient attribute values range is required.
+
+To get minimum and maximum attribute values [statistics](https://github.com/Esri/i3s-spec/blob/master/docs/1.9/statisticsInfo.cmn.md) can be used. The [statistics info JSON](https://github.com/Esri/i3s-spec/blob/master/docs/1.9/statsInfo.cmn.md) has min and max values. Usage of those values allows setting true attribute values range not clamping extremum values.
+
+As soon as statistics info is stored in separate resources, it has to be loaded in a separate request. Statistics is just a JSON data and can be loaded the following way:
+
+```ts
+import {fetchFile} from '@loaders.gl/core';
+import {StatsInfo} from '@loaders.gl/i3s';
+
+const dataResponse = await fetchFile(`${url}/statistics/f_5/0`);
+const data = JSON.parse(await dataResponse.text());
+const stats: StatsInfo | null = (data?.stats as StatsInfo) || null;
+```
+
+## Use @deck.gl-community/layers

```

Comment:
Maybe the section should be called `DataDrivenTile3DLayer` - the fact that it is in the deck.gl-community repo/namespace is more of a detail?

If we reference this layer, does it have documentation in the deck.gl-community website that we can link to?

Reply:
You right, changed to DataDrivenTile3DLayer.
There is no documentation in the deck.gl-community for the layer.